python/pip_install/extract_wheels/lib/bazel.py

```diff
@@ -147,6 +147,7 @@ def generate_build_file_contents(
                 "**/* *",
                 "**/*.py",
                 "**/*.pyc",
+                "*dist-info/**",
```

Comment:
I've applied this patch but think it would be good to know the packages. I'm also not necessarily advocating for excluding all of `dist-info`, I just don't know of a case where it's needed and I know it contains non-deterministic data. Happy to go with something more focused for now. I leave it up to you.

Reply:
I'll dig them out next week. I think this is likely fine to merge as is. Can always revert or update if it causes issues.